src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
```

Comment:
you'll not use background-image since we'll use `img` tag

Reply:
not fixed
https://github.com/mate-academy/layout_product-cards/pull/3872/files#r1414357751